### DISCUSSION TITLE
Service could not start due to logon failure

### DISCUSSION THREAD

**Discussion by Daitallica** [sandboxie-plus/Sandboxie#2693] [Q&A Feedback]
Hi

New to sandboxie and didn't know where to come for help.

I have installed a program that needs a service to work properly,
the service is installed in the sandbox but will not run due to a logon failure.
It won't work as a local system account,
when I try as my user account I get the error
"my username" could not be granted the log on as a service right.

Is there a "sandboxie user" I can run it as, or is there a certain way to get the service to work?

Thank you

**Comment by bastik-1001** [comment]
There is no "sandboxie user". If you are talking about SandboxieLogon, this referrers to how sandboxed processes are run. Without SandboxieLogon=y all sandboxes will be run as the same ANONYMOUS, whereas SandboxieLogon=y isolates them by the sandbox name with an individual SID.

If this problem still exists with the latest release (or even pre-releases) I suggest telling us, what software is affected. Maybe, this is even an issue with Sandboxie, or they are incompatible with each other.
